pex/common.py

```diff
@@ -20,11 +20,11 @@
 
 
 # We use the start of MS-DOS time, which is what zipfiles use (see section 4.4.6 of
-# https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT). ZipInfo expects a 6-tuple of
-# year-month-day-hour-minute-second, which we use here.
+# https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT). Note that ZipInfo expects a
+# time.struct_time, which we set here.
```

Comment:
But now this is a lie. `PermPreservingZipFile.zip_info_from_file` expects a `time.struct_time`. ZipInfo expects a 6-tuple of year-month-day-hour-minute-second.